external_search.py

```diff
@@ -446,16 +446,9 @@ def bulk_update(self, works, retry_on_batch_failure=True):
         needs_add = []
         successes = []
         for work in works:
-            if work.presentation_ready:
-                needs_add.append(work)
-            else:
-                # Works are removed one at a time, which shouldn't
-                # pose a performance problem because works almost never
-                # stop being presentation ready.
-                self.remove_work(work)
-                successes.append(work)
+            needs_add.append(work)
```

Comment:
This change is not strictly related to this branch; it could have been made before but I didn't notice earlier. We now store works that are not presentation-ready in the search index, so `bulk_update` should include them rather than trying to remove them.